pallets/kensetsu/Cargo.toml

```diff
@@ -42,6 +42,7 @@ assets = { path = "../assets", default-features = false }
 [features]
 default = ["std"]
 std = [
+	"assets/std",
```

Comment:
Is it necessary if you use `assets` in `dev-dependecies` only?

Reply:
Yes, tests require `std` feature on assets.